compiler/passes/scopeResolve.cpp

```diff
@@ -1588,13 +1601,20 @@ void checkConflictingSymbols(std::vector<Symbol *>& symbols,
                      failedUSymExprs.end(),
                      context) == 0) {
         failedUSymExprs.push_back(context);
-        astlocT* symRenameLoc = renameLocs[sym];
         USR_FATAL_CONT(sym, "symbol %s is multiply defined", name);
+
+        if (VisibilityStmt* reexport = reexportPts[sym]) {
+          USR_PRINT("'%s' was reexported at %s:%d, which impacts its "
```

Comment:
do we need the part about closeness?

Reply:
I thought it clarified why we cared about the re-export point, but I can definitely drop it

Reply:
I don't have a strong preference. Left to my own devices, I'd drop it. I think I find it a bit too... operational (?) for a typical user.